### ISSUE TITLE
Possible way to move file/folder

### ISSUE THREAD

**Issue by Antexa** [mmozeiko/RcloneBrowser#13]
Hi, 
It would be nice if we get the ability of moving file/folder to another folder

**Comment by mmozeiko** [comment]
Technically its possible - you need to go to source file or folder and press "Download" for it. Then in destination simply type same-remote-name:/destination/folder. Destination must be a folder. If source is a file, then it will will be placed in destination folder with same file name (I am not yet using new rclone copyto and moveto commands). Destination folder will be created if it doesn't exist.

Not a very nice way, but it works. I couldn't figure out nice way to allow this in GUI.

**Comment by dewa710** [comment]
You could add Cut and Paste options to the right click popup, or move files via mouse drag

**Comment by amaklp** [comment]
+1000
It would be great to have the ability to easily move files and folders, especially for our encrypted remotes because we can't figure out which file is which so we can't move them from Amazons graphical interface.

**Comment by mmozeiko** [comment]
Well actual moving will be responsibility of rclone. I'll provide only easy way to start this from GUI.
Afaik currently rclone cannot do move inside crypt remotes. It will download and upload file. Not sure if that is fixed yet or not.

**Comment by amaklp** [comment]
Thanks for the reply @mmozeiko !
> I'll provide only easy way to start this from GUI.

Yes! That's what I meant.

I haven't tested the move command on an ecnrypted remote, but I don't see any limitation for encrypted remotes in the documentation.
http://rclone.org/commands/rclone_move/

**Comment by dewa710** [comment]
@mmozeiko 
the moveto command works on crypt remotes without downloading/uploading the files

**Comment by amaklp** [comment]
Oh right. So probably using this command on the GUI will work :)

**Comment by mmozeiko** [comment]
@dewa710, excellent. I'll switch to use moveto instead of move.

**Comment by vampywiz17** [comment]
You will plan to add "right click" menu to move files with same remote?

**Comment by CottonEaster** [comment]
This is just to say that I too would welcome the ability to move folders (and let me add: thank you very much for rclone-browser; it is something of a godsend).

**Comment by NoLooseEnds** [comment]
Yes, would be nice to able to move folders via the GUI on the remote.

**Comment by amlopin** [comment]
Any update of this?

**Comment by wingsuit** [comment]
Did implementing the moveTo command on right click prove tricky or is there a backlog of other features in front of this one?

**Comment by m4rii0** [comment]
Any update on this?
